### ISSUE TITLE
Example command 

### ISSUE THREAD

**Issue by zeroeskeys** [naksyn/Pyramid#12]
The given example command 
`python3 pyramid.py -p 443 -ssl -u testuser -pass Sup3rP4ss! -enc "chacha20" -passenc "TestPass1" -server "192.168.1.2" -generate -setcradle pythonmemorymodule.py` 

Appears to cause issues due to the quotes shown. 
The Python REPL crashes out and you get a non-execution of the payload.

In testing I omit these double quotes around the pass, server and enc and I have no trouble, works very well after that. 
I note in the video that you do not use the quotes.

**Comment by naksyn** [comment]
I cannot replicate your error. As you can see from the screenshot below, issuing the command you wrote correctly starts the server.
![immagine](https://github.com/naksyn/Pyramid/assets/59816245/ae424e9b-8d5e-4f69-ac37-7c56d6cab46f)

**Comment by zeroeskeys** [comment]
Apologies, I wasn't clear on when the issue happens. The issue I experienced was after using the example command to generate the paste command, when you paste that generated command into a python repl on the client/target side where you want to execute is where it was crashing. 

However I have tested again and it's not consistent with the quotes strings, I think it may be when using embedded 3.12.2 I get this issue, I may have conflated during testing where I was using installed python to test and also using the embedded 3.12.2 version. Installed Python seems fine with higher versions client side, embedded does not seem happy above 3.10.11 (*not tested other versions above 3.10.11*).

Using the embedded 3.10.11 I don't have the issue. Sorry for the trouble. 

![image](https://github.com/naksyn/Pyramid/assets/29241769/b86fdeab-f9a4-481f-9bdd-1a413f4f42b3)
_3.12.2 embedded crashes_

![image](https://github.com/naksyn/Pyramid/assets/29241769/3c1905f7-3397-424c-baa5-f4fa901592e2)
_3.10.11 embedded is fine_
